pom.xml

```
@@ -32,6 +32,7 @@
     <inceptionYear>2018</inceptionYear>
     <properties>
         <main-class>nl.knaw.dans.easy.validatebag.Command</main-class>
+        <easy.licenses.version>1.0.1-SNAPSHOT</easy.licenses.version>
```

```suggestion
        <easy-licenses.version>1.0.1-SNAPSHOT</easy-licenses.version>
```
This is the convention used everywhere else.